### ISSUE TITLE
Don't see any way to access menu

### ISSUE THREAD

**Issue by stevecj** [ringcentral/ringcentral-community-app#27]
I notice that ability to see currently installed version number was added via about in the menu, but I don't see any way to get to an app-level menu.

**Comment by embbnux** [comment]
Yeah, electron menu doesn't show at gnome. Let me check how to show that. 
For now, you can get version: In app -> Resource Center -> Get app info

**Comment by embbnux** [comment]
Hi @stevecj for menubar issue, can you try to press Alt key to see if you can get menubar? It seems some settings in system. In our Ubuntu 20.04, most apps don't have menubar. Only Firefox can get that by right click to show it.

BTW I only add about menu at Tray.

**Comment by stevecj** [comment]
Alt key did not work for me, but app -> Resource Center -> Get app info does, and I'm happy with that.
